devicetypes/smartthings/zigbee-dimmer-with-motion-sensor.src/zigbee-dimmer-with-motion-sensor.groovy

```diff
@@ -21,6 +21,7 @@ metadata {
 		capability "Light"
 		capability "Switch"
 		capability "Switch Level"
+		capability "Power Meter"
```

Comment:
For consistency with `zigbee-dimmer` and `zigbee-dimmer-power` we might want to change this to `zigbee-dimmer-power-with-motion`

Reply:
It doesn't look like this has gone to prod yet so I think we should change it now.